### PR TITLE
Added note about tenant with RHM enabled

### PR DESCRIPTION
**Pull request by dilip-2201**
Added below note:
Note: If the Tenant is enabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be Enabled in all regions. You must execute this cmdlet in each region that are added in Allowed Data Location for Skype for Business.

### CODE REVIEW

File: skype/skype-ps/skype/Enable-CsOnlineSipDomain.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ Before running this cmdlet for any SIP domain in a Skype for Business Server dep
 **IMPORTANT**
 If you have more than one on-premises deployment of Skype for Business Server, you *must* ensure SharedSipAddressSpace is disabled in all other Skype for Business Server deployments except the deployment containing the SIP domain that is being enabled. 
 
+Note: If the Tenant is enabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be Enabled in all regions. You must execute this cmdlet in each region that are added in Allowed Data Location for Skype for Business.
```
Comment (dariomws) [inline skype/skype-ps/skype/Enable-CsOnlineSipDomain.md:37]:
```suggestion
Note: If the Tenant is enabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be Enabled in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location for Skype for Business.  
```

**Review by dariomws — APPROVED** [review]

**Comment by get-itips** [comment]
Isn't this conflicting with what is being said in this other PR ? https://github.com/MicrosoftDocs/office-docs-powershell/pull/7520/files

**Comment by dilip-2201** [comment]
@get-itips - These are 2 different documents which I had requested PR one cmdlet for enable-csonlinesipdomain and other one for disable-csonlinesipdomain.

**Comment by get-itips** [comment]
> @get-itips - These are 2 different documents which I had requested PR one cmdlet for enable-csonlinesipdomain and other one for disable-csonlinesipdomain.

Yes, but aren't the two notes conflicting? both start with "if the tenant is enabled..."

On Disable-

> Note: If the Tenant is **enabled** for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be **disabled** in all regions. You must execute this cmdlet in each region that are added in Allowed Data Location.

On Enable-

> Note: If the Tenant is **enabled** for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be **Enabled** in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location for Skype for Business.

File: skype/skype-ps/skype/Enable-CsOnlineSipDomain.md [hunk 2]
```diff
@@ -34,6 +34,8 @@ Before running this cmdlet for any SIP domain in a Skype for Business Server dep
 **IMPORTANT**
 If you have more than one on-premises deployment of Skype for Business Server, you *must* ensure SharedSipAddressSpace is disabled in all other Skype for Business Server deployments except the deployment containing the SIP domain that is being enabled. 
 
+Note: If the Tenant is enabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be Enabled in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location for Skype for Business.
```
Comment (dariomws) [inline skype/skype-ps/skype/Enable-CsOnlineSipDomain.md:37]:
```suggestion
Note: If the Tenant is disabled for Regionally Hosted Meetings in Skype for Business Online, Online SIP Domains must be Enabled in all regions. You must execute this cmdlet in each region that is added in Allowed Data Location for Skype for Business.  
```

**Review by dariomws — COMMENTED** [review]
Likelihood here it should be "if disabled, sip domains must be disabled in all regions".

File: skype/skype-ps/skype/Enable-CsOnlineSipDomain.md (hunk 2 above)
Comment (dariomws) [reply skype/skype-ps/skype/Enable-CsOnlineSipDomain.md:37]: Sorry, this comment is for the other PR.

Comment (dilip-2201) [reply skype/skype-ps/skype/Enable-CsOnlineSipDomain.md:37]: If the Tenant is enabled for RHM in SFBO - 
either running enable or disable sip domain command - must be run on all the forests to reflect the changes correctly.
Check the last warning in the screenshot:
image

**Comment by dilip-2201** [comment]
@get-itips @dariomws 
If the Tenant is enabled for RHM in SFBO -
either running enable or disable sip domain command - must be run on all the forests to reflect the changes correctly.
Check the last warning in the screenshot:
image

**Comment by dariomws** [comment]
@dilip-2201 Ok, now reading the warning, I think I understand. If this tenant is enabled for RHM, SIP domains must be enabled or disabled in all regions (not forests).

But what I do not understand/know is how you can run this cmdlet on different regions.

Maybe we can leave this note as it's in the warning.

**Comment by dilip-2201** [comment]
@dariomws  - 
Agreed, we can use the same warning in cmdlet. Without the region names, make it generic.
________________
Fyi - Customers can connect using specific Powershell URI for each region (forest). RHM enabled customers usually will have this information.

this command will show the forests/region enabled for all O365 services including SfBO:
get-cstenant| Select -ExpandProperty alloweddatalocation

**Comment by get-itips** [comment]
Thanks for the clarification @dilip-2201

**Review by dariomws — APPROVED** [review]
@dilip-2201 my apologies, now with your comments I understand. And the note you added is practically like the warning (without specific regions). I approve it again just in case to avoid confusions.

Thank you so much for your explanation!

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne

**Pull request merged**